ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/commons/RaoUtil.java

```diff
@@ -251,4 +255,9 @@ public static boolean cnecShouldBeOptimized(Map<FlowCnec, RangeAction<?>> flowCn
 
         return cnecMarginToUpperBound + maxFlowDecrease < 0 || cnecMarginToLowerBound + maxFlowIncrease < 0;
     }
+
+    public static void applyRemedialActions(Network network, OptimizationResult optResult, State state) {
```

Comment:
Does this really need to be in here rather than in AutomatonSimulator?

Reply:
Yes because it is also used in CastorFullOpt for preventive and curative states